### ISSUE TITLE
Page drawer implementation enhancement

### ISSUE THREAD

**Issue by mattnolting** [RedHatInsights/insights-chrome#1279]
Current structure is:

```
.pf-c-page
  .pf-c-page__header
  .pf-c-page__sidebar
  .pf-c-page__drawer
    .pf-c-drawer__content
```

This causes a height issue as `.pf-c-drawer` defines that, and it's missing.

Is should be:

```
.pf-c-page
  .pf-c-page__header
  .pf-c-page__sidebar
  .pf-c-page__drawer
    .pf-c-drawer <---- insert drawer
      .pf-c-drawer__main <---- insert drawer main
        .pf-c-drawer__content
```

Making this change should eliminate the need for https://github.com/RedHatInsights/insights-chrome/blob/master/src/sass/chrome.scss#L91, which causes issues in various apps..

**Comment by Hyperkid123** [comment]
@mattnolting I agree that the layout is quite strange now. However, we now face the "problem" that we need to support old applications in chrome 2 which require a slightly different layout. For example, we have two `.pf-c-page` elements in chrome 2 apps which is obviously not ideal, but we need it in order to keep the support for apps that are not yet migrated to chrome 2.

We first have to wait for all apps to be migrated to the chrome 2 build. And I don't think that will happen before the summit. But I will add it to the list of post-summit cleanup.

**Comment by mattnolting** [comment]
@Hyperkid123 Awesome, thanks!

**Comment by Hyperkid123** [comment]
@mattnolting we can finally deal with these DOM issues. The DOM structure should be fine not. But we have more than one drawer ATM. One is for quickstart and one for help topics. I am trying to communicate and find a solution with the PF quickstart devs.
